opensrp-giz-malawi/build.gradle

```diff
@@ -158,7 +158,7 @@ dependencies {
         exclude group: 'io.ona.rdt-capture', module: 'lib'
     }
 
-    implementation('org.smartregister:opensrp-client-child:0.2.0-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-child:0.2.1-SNAPSHOT@aar') {
```

Comment:
The following issues do not work:
- **Fixed adverse effect report date constraint**
- **Added Child's Registration Number field** - I can enter alphanumeric but it just validates (Change this to enable the numpad automatically)

The following works:
- **Added mother tdv doses field**

The assembled app is from commit `d4cb896e730e49b98de6b1be7898d3f939fe3571`

Reply:
- **Added Child's Registration Number field** - I can enter alphanumeric but it just validates (Change this to enable the numpad automatically) **Fixed this**